src/scvi/external/mrvi/_model.py

```diff
@@ -995,21 +996,32 @@ def get_outlier_cell_sample_pairs(
             adata_s = adata[sample_idxs]
 
             ap = self.get_aggregated_posterior(adata=adata, indices=sample_idxs)
```

Comment:
Could you quickly update the docstring for this method?
Also, could you remind me what is the purpose of `threshs` again?

Reply:
its a threshold that determines if a cell-sample pair is admissible or not